fea-rs/src/compile/lookups/gpos_builders.rs

```diff
@@ -540,6 +541,10 @@ pub struct MarkToLigBuilder {
 }
 
 impl MarkToLigBuilder {
+    /// Add a new mark glyph.
+    ///
+    /// If this glyph already exists in another mark class, we return the
+    /// previous class; this is likely an error.
```

Comment:
Is that what fontmake does? If not suggest filing an issue about this so we don't forget.